web_src/js/features/common-global.js

```diff
@@ -89,9 +89,12 @@ export function initGlobalCommon() {
 
   // Semantic UI modules.
   const $uiDropdowns = $('.ui.dropdown');
-  $uiDropdowns.filter(':not(.custom)').dropdown({
-    fullTextSearch: 'exact'
-  });
+
+  // do not init "custom" dropdowns, "custom" dropdowns are managed by their own code.
+  $uiDropdowns.filter(':not(.custom)').dropdown({fullTextSearch: 'exact'});
+
+  // "jump" means this dropdown is mainly used for "menu" purpose, clicking an item will jump to somewhere else or trigger an action/function.
+  // when a dropdown is used for non-refresh actions with tippy, it must have this "jump" class to hide the tippy when dropdown is closed.
```

Comment:
Can we wrap such comments to something more pleasant, like 100 chars? This is 140 chars and too much for my screen :)

Reply:
done https://github.com/go-gitea/gitea/pull/23450/commits/15285595298994035f4d5e0aba1c44d026c4550b